pkgs/development/libraries/libxslt/default.nix

```diff
@@ -32,6 +32,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     pkg-config
     autoreconfHook
+  ] ++ lib.optionals cryptoSupport [
+    libgcrypt
```

Comment:
We should add a short comment why this is added here

Reply:
It should not be there in the first place.

Reply:
This might work for an alternative: https://github.com/NixOS/nixpkgs/blob/6b251b07d2800831cf84924b69939aeeb11de763/pkgs/by-name/li/libgnome-keyring/package.nix#L20-L24

Reply:
Cool, that even works in `pkgsCross.aarch64-multiplatform.libxslt`!